### ISSUE TITLE
Wrong results with video file!

### ISSUE THREAD

**Issue by omarrajab94** [OlafenwaMoses/Traffic-Net#3]
I get wrong results when i test the model with video file, but not, if i give pictures to the model. can you please show me, how can i make prediction for video frames.
results look like:
Fire  :  100.0
Sparse_Traffic  :  0.0
Dense_Traffic  :  0.0
Accident  :  0.0
video file is: https://www.pexels.com/video/traffic-flow-in-the-highway-2103099/

my Code: 
<img width="707" alt="Bildschirmfoto 2020-01-12 um 20 41 53" src="https://user-images.githubusercontent.com/49998197/72224562-1c958800-357c-11ea-87ce-c8956740ec1d.png">

**Comment by cross-hello** [comment]
That is right, with a wrong result is normal behavior. The actual life view may differ from input photos, which mean the perspective angle, car size may different from one by one. you need to train the model by your inputs.
